From: Hilko Bengen <bengen@debian.org>
Date: Thu, 30 Dec 2021 21:05:43 +0100
Subject: Set special CPU type on RISCV64

---
 lib/appliance-cpu.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/lib/appliance-cpu.c b/lib/appliance-cpu.c
index 54ac6e2..1f50ffa 100644
--- a/lib/appliance-cpu.c
+++ b/lib/appliance-cpu.c
@@ -84,6 +84,8 @@ guestfs_int_get_cpu_model (int kvm)
 #elif defined(__powerpc64__)
   /* See discussion in https://bugzilla.redhat.com/show_bug.cgi?id=1605071 */
   return NULL;
+#elif defined(__riscv)
+  return "rv64";
 #else
   /* On most architectures we can use "max" to get the best possible CPU.
    * For recent qemu this should work even on TCG.
