From: Hilko Bengen <bengen@debian.org>
Date: Sat, 26 Nov 2022 16:12:08 +0100
Subject: Fix include path for generating localized manpages

---
 po-docs/language.mk | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/po-docs/language.mk b/po-docs/language.mk
index 2d5f0fe..0135be0 100644
--- a/po-docs/language.mk
+++ b/po-docs/language.mk
@@ -107,6 +107,7 @@ guestfish.1: guestfish.pod guestfish-actions.pod guestfish-commands.pod guestfis
 	$(PODWRAPPER) \
 	  --no-strict-checks \
 	  --man $@ \
+	  --path $(srcdir) \
 	  --license GPLv2+ \
 	  $<
 
@@ -149,6 +150,7 @@ virt-p2v.1: virt-p2v.pod virt-p2v-kernel-config.pod
 	$(PODWRAPPER) \
 	  --no-strict-checks \
 	  --man $@ \
+	  --path $(srcdir) \
 	  $<
 
 %.3: %.pod
